### PR TITLE
Rules update 1

### PR DESCRIPTION
**Pull request by tealtan**
Hi @lisawray,

I’ve added new rules for headline that should cover most cases.

Still outstanding:
- [ ] Medium viewport (waiting until some refinements are finalized before putting these in)
- [ ] Video card at all sizes (waiting until some refinements have been made by the video team)

Your note about using a text style called `hidden` sounds good, I’ve added that for `status`.

Can you run your test based on these new rules to see if we have headline covered? (minus the above)
I also noticed that the test generates a number of duplicates – it would be cool if we can dedupe them, but I can also do that easily so no urgency there.

Thanks!

### CODE REVIEW

**Comment by lisawray** [comment]
@tealtan 
Can confirm we are 99% there on headlines (minus video and medium viewport!)

Apart from the above, we're still missing:

```
Missing text style definition for: l_headline_large_urgent
```

Headline:
```
Missing rule for URGENT Whole hard 4-column card in large viewport
Missing rule for URGENT Whole soft 4-column card in large viewport
```

**Comment by lisawray** [comment]
@tealtan i'm still seeing lots of missing coverage in summary and photo credit -- i assume you're on it but let me know if the print outs would be helpful

**Comment by tealtan** [comment]
@lisawray Thanks! This commit should take care of those missing urgent ones.

I’m taking a first pass at summary and photo credit – I’m ok for now, but a check will be helpful later, for sure!

**Comment by tealtan** [comment]
Hi @lisawray,

Could you run a check on summary coverage? The last commit here should address a lot of the missing rules for it. Thanks!

File: card_design_rules.json [hunk 1]
```diff
@@ -932,35 +913,46 @@
...
+          "visual"
+        ],
         "rendition": [
           "skim",
           "whole"
         ],
-        "viewport": [
-          "large"
+        "text_style": "hidden"
+      }
```
Comment (lisawray) [inline card_design_rules.json]: you're missing a comma right here!

**Comment by tealtan** [comment]
Ah, thank you!

**Comment by lisawray** [comment]
I'm still working on getting you test results -- should be today!  Thanks for patience

**Comment by tealtan** [comment]
NP! ✌️

**Comment by lisawray** [comment]
Here's what's missing for summary:
```
Missing rule for TEXT Skim hard 1-column card in small viewport
Missing rule for TEXT Skim hard 1-column card in large viewport
Missing rule for TEXT Skim hard 2-column card in small viewport
Missing rule for TEXT Skim hard 2-column card in large viewport
Missing rule for TEXT Skim hard 3-column card in small viewport
Missing rule for TEXT Skim hard 3-column card in large viewport
Missing rule for TEXT Skim hard 4-column card in small viewport
Missing rule for TEXT Skim hard 4-column card in large viewport
Missing rule for TEXT Skim soft 1-column card in small viewport
Missing rule for TEXT Skim soft 1-column card in large viewport
Missing rule for TEXT Skim soft 2-column card in small viewport
Missing rule for TEXT Skim soft 2-column card in large viewport
Missing rule for TEXT Skim soft 3-column card in small viewport
Missing rule for TEXT Skim soft 3-column card in large viewport
Missing rule for TEXT Skim soft 4-column card in small viewport
Missing rule for TEXT Skim soft 4-column card in large viewport
Missing rule for TEXT Whole hard 1-column card in large viewport
Missing rule for TEXT Whole hard 2-column card in large viewport
Missing rule for TEXT Whole hard 3-column card in large viewport
Missing rule for TEXT Whole hard 4-column card in large viewport
Missing rule for TEXT Whole soft 1-column card in large viewport
Missing rule for TEXT Whole soft 2-column card in large viewport
Missing rule for TEXT Whole soft 3-column card in large viewport
Missing rule for TEXT Whole soft 4-column card in large viewport
Missing rule for VIDEO Skim hard 1-column card in small viewport
Missing rule for VIDEO Skim hard 1-column card in large viewport
Missing rule for VIDEO Skim hard 2-column card in small viewport
Missing rule for VIDEO Skim hard 2-column card in large viewport
Missing rule for VIDEO Skim hard 3-column card in small viewport
Missing rule for VIDEO Skim hard 3-column card in large viewport
Missing rule for VIDEO Skim hard 4-column card in small viewport
Missing rule for VIDEO Skim hard 4-column card in large viewport
Missing rule for VIDEO Skim soft 1-column card in small viewport
Missing rule for VIDEO Skim soft 1-column card in large viewport
Missing rule for VIDEO Skim soft 2-column card in small viewport
Missing rule for VIDEO Skim soft 2-column card in large viewport
Missing rule for VIDEO Skim soft 3-column card in small viewport
Missing rule for VIDEO Skim soft 3-column card in large viewport
Missing rule for VIDEO Skim soft 4-column card in small viewport
Missing rule for VIDEO Skim soft 4-column card in large viewport
Missing rule for VIDEO Whole hard 1-column card in small viewport
Missing rule for VIDEO Whole hard 1-column card in large viewport
Missing rule for VIDEO Whole hard 2-column card in small viewport
Missing rule for VIDEO Whole hard 2-column card in large viewport
Missing rule for VIDEO Whole hard 3-column card in small viewport
Missing rule for VIDEO Whole hard 3-column card in large viewport
Missing rule for VIDEO Whole hard 4-column card in small viewport
Missing rule for VIDEO Whole hard 4-column card in large viewport
Missing rule for VIDEO Whole soft 1-column card in small viewport
Missing rule for VIDEO Whole soft 1-column card in large viewport
Missing rule for VIDEO Whole soft 2-column card in small viewport
Missing rule for VIDEO Whole soft 2-column card in large viewport
Missing rule for VIDEO Whole soft 3-column card in small viewport
Missing rule for VIDEO Whole soft 3-column card in large viewport
Missing rule for VIDEO Whole soft 4-column card in small viewport
Missing rule for VIDEO Whole soft 4-column card in large viewport
Missing rule for SLIDESHOW Skim hard 1-column card in small viewport
Missing rule for SLIDESHOW Skim hard 1-column card in large viewport
Missing rule for SLIDESHOW Skim hard 2-column card in small viewport
Missing rule for SLIDESHOW Skim hard 2-column card in large viewport
Missing rule for SLIDESHOW Skim hard 3-column card in small viewport
Missing rule for SLIDESHOW Skim hard 3-column card in large viewport
Missing rule for SLIDESHOW Skim hard 4-column card in small viewport
Missing rule for SLIDESHOW Skim hard 4-column card in large viewport
Missing rule for SLIDESHOW Skim soft 1-column card in small viewport
Missing rule for SLIDESHOW Skim soft 1-column card in large viewport
Missing rule for SLIDESHOW Skim soft 2-column card in small viewport
Missing rule for SLIDESHOW Skim soft 2-column card in large viewport
Missing rule for SLIDESHOW Skim soft 3-column card in small viewport
Missing rule for SLIDESHOW Skim soft 3-column card in large viewport
Missing rule for SLIDESHOW Skim soft 4-column card in small viewport
Missing rule for SLIDESHOW Skim soft 4-column card in large viewport
Missing rule for SLIDESHOW Whole hard 1-column card in small viewport
Missing rule for SLIDESHOW Whole hard 1-column card in large viewport
Missing rule for SLIDESHOW Whole hard 2-column card in small viewport
Missing rule for SLIDESHOW Whole hard 2-column card in large viewport
Missing rule for SLIDESHOW Whole hard 3-column card in small viewport
Missing rule for SLIDESHOW Whole hard 3-column card in large viewport
Missing rule for SLIDESHOW Whole hard 4-column card in small viewport
Missing rule for SLIDESHOW Whole hard 4-column card in large viewport
Missing rule for SLIDESHOW Whole soft 1-column card in small viewport
Missing rule for SLIDESHOW Whole soft 1-column card in large viewport
Missing rule for SLIDESHOW Whole soft 2-column card in small viewport
Missing rule for SLIDESHOW Whole soft 2-column card in large viewport
Missing rule for SLIDESHOW Whole soft 3-column card in small viewport
Missing rule for SLIDESHOW Whole soft 3-column card in large viewport
Missing rule for SLIDESHOW Whole soft 4-column card in small viewport
Missing rule for SLIDESHOW Whole soft 4-column card in large viewport
```

**Comment by tealtan** [comment]
Cool, thanks! This last commit should hopefully take care of those rules.

**Comment by tealtan** [comment]
Hi @lisawray,
I’ve reordered a few rules in the last commit and, more importantly, have added rules for medium! Can you run a check on the headline element and see where we’re falling short? Thanks!

**Comment by lisawray** [comment]
Here's headline: 
```
Missing rule for STANDARD Whole hard 1-column card in medium viewport
Missing rule for STANDARD Whole hard 2-column card in medium viewport
Missing rule for STANDARD Whole hard 3-column card in medium viewport
Missing rule for STANDARD Whole hard 4-column card in medium viewport
Missing rule for STANDARD Whole soft 1-column card in medium viewport
Missing rule for STANDARD Whole soft 2-column card in medium viewport
Missing rule for STANDARD Whole soft 3-column card in medium viewport
Missing rule for STANDARD Whole soft 4-column card in medium viewport
Missing rule for BULLET Whole hard 1-column card in medium viewport
Missing rule for BULLET Whole hard 2-column card in medium viewport
Missing rule for BULLET Whole hard 3-column card in medium viewport
Missing rule for BULLET Whole hard 4-column card in medium viewport
Missing rule for BULLET Whole soft 1-column card in medium viewport
Missing rule for BULLET Whole soft 2-column card in medium viewport
Missing rule for BULLET Whole soft 3-column card in medium viewport
Missing rule for BULLET Whole soft 4-column card in medium viewport
Missing rule for VISUAL Whole hard 1-column card in medium viewport
Missing rule for VISUAL Whole hard 2-column card in medium viewport
Missing rule for VISUAL Whole hard 3-column card in medium viewport
Missing rule for VISUAL Whole hard 4-column card in medium viewport
Missing rule for VISUAL Whole soft 1-column card in medium viewport
Missing rule for VISUAL Whole soft 2-column card in medium viewport
Missing rule for VISUAL Whole soft 3-column card in medium viewport
Missing rule for VISUAL Whole soft 4-column card in medium viewport
Missing rule for TEXT Whole hard 1-column card in medium viewport
Missing rule for TEXT Whole hard 2-column card in medium viewport
Missing rule for TEXT Whole hard 3-column card in medium viewport
Missing rule for TEXT Whole hard 4-column card in medium viewport
Missing rule for TEXT Whole soft 1-column card in medium viewport
Missing rule for TEXT Whole soft 2-column card in medium viewport
Missing rule for TEXT Whole soft 3-column card in medium viewport
Missing rule for TEXT Whole soft 4-column card in medium viewport
Missing rule for EXCERPT Whole hard 1-column card in medium viewport
Missing rule for EXCERPT Whole hard 2-column card in medium viewport
Missing rule for EXCERPT Whole soft 1-column card in medium viewport
Missing rule for EXCERPT Whole soft 2-column card in medium viewport
Missing rule for AUTHOR Whole hard 1-column card in medium viewport
Missing rule for AUTHOR Whole hard 2-column card in medium viewport
Missing rule for AUTHOR Whole hard 3-column card in medium viewport
Missing rule for AUTHOR Whole hard 4-column card in medium viewport
Missing rule for AUTHOR Whole soft 1-column card in medium viewport
Missing rule for AUTHOR Whole soft 2-column card in medium viewport
Missing rule for AUTHOR Whole soft 3-column card in medium viewport
Missing rule for AUTHOR Whole soft 4-column card in medium viewport
Missing rule for VIDEO Headline hard 1-column card in small viewport
Missing rule for VIDEO Headline hard 2-column card in small viewport
Missing rule for VIDEO Headline hard 3-column card in small viewport
Missing rule for VIDEO Headline hard 4-column card in small viewport
Missing rule for VIDEO Headline soft 1-column card in small viewport
Missing rule for VIDEO Headline soft 2-column card in small viewport
Missing rule for VIDEO Headline soft 3-column card in small viewport
Missing rule for VIDEO Headline soft 4-column card in small viewport
Missing rule for VIDEO Whole hard 1-column card in small viewport
Missing rule for VIDEO Whole hard 1-column card in medium viewport
Missing rule for VIDEO Whole hard 1-column card in large viewport
Missing rule for VIDEO Whole hard 2-column card in small viewport
Missing rule for VIDEO Whole hard 2-column card in medium viewport
Missing rule for VIDEO Whole hard 2-column card in large viewport
Missing rule for VIDEO Whole hard 3-column card in small viewport
Missing rule for VIDEO Whole hard 3-column card in medium viewport
Missing rule for VIDEO Whole hard 3-column card in large viewport
Missing rule for VIDEO Whole hard 4-column card in small viewport
Missing rule for VIDEO Whole hard 4-column card in medium viewport
Missing rule for VIDEO Whole hard 4-column card in large viewport
Missing rule for VIDEO Whole soft 1-column card in small viewport
Missing rule for VIDEO Whole soft 1-column card in medium viewport
Missing rule for VIDEO Whole soft 1-column card in large viewport
Missing rule for VIDEO Whole soft 2-column card in small viewport
Missing rule for VIDEO Whole soft 2-column card in medium viewport
Missing rule for VIDEO Whole soft 2-column card in large viewport
Missing rule for VIDEO Whole soft 3-column card in small viewport
Missing rule for VIDEO Whole soft 3-column card in medium viewport
Missing rule for VIDEO Whole soft 3-column card in large viewport
Missing rule for VIDEO Whole soft 4-column card in small viewport
Missing rule for VIDEO Whole soft 4-column card in medium viewport
Missing rule for VIDEO Whole soft 4-column card in large viewport
Missing rule for SLIDESHOW Whole hard 1-column card in medium viewport
Missing rule for SLIDESHOW Whole hard 2-column card in medium viewport
Missing rule for SLIDESHOW Whole hard 3-column card in medium viewport
Missing rule for SLIDESHOW Whole hard 4-column card in medium viewport
Missing rule for SLIDESHOW Whole soft 1-column card in medium viewport
Missing rule for SLIDESHOW Whole soft 2-column card in medium viewport
Missing rule for SLIDESHOW Whole soft 3-column card in medium viewport
Missing rule for SLIDESHOW Whole soft 4-column card in medium viewport
```
